src/models/WidgetEventCapability.ts

```diff
@@ -19,6 +19,7 @@ import { Capability } from "..";
 export enum EventDirection {
     Send = "send",
     Receive = "receive",
```

Comment:
Hmm, could this concept be named `listen` instead...? "Receive" and "read" are quite similar... I guess it needs an MSC to change the capabilities though... 😅 

If it's too late to actually rename, then at least add some comments... I have already been unsure each time I saw either one while reading this PR.

Reply:
We can rename the enum name, but the value would ideally stay the same (or be changed in that MSC, which hasn't landed yet, but it would require managing a tech debt episode we don't have availability for)

Reply:
hmm, actually it's not safe to rename the enum at this stage, but we should probably do that before a proper release of this SDK... Have filed https://github.com/matrix-org/matrix-widget-api/issues/35 to track